test/library/packages/LinearAlgebra/performance/norm-perf.perfexecopts

```diff
@@ -0,0 +1,2 @@
+--m=1000 --iters=100 #norm-1000x1000
```

Comment:
I would suggest bumping the problem size by an order of magnitude or 2, and reducing the iteration time so that our timings we record are not so small, e.g.

```
--m=10000 --iters=32 #norm-1000x1000
--m=100000 --iters=32 --testMatrixNorm=false #norm-1000
```